www/src/pages/docs/ssr-apis.js

```diff
@@ -65,5 +76,21 @@ export const pageQuery = graphql`
         ...DocumentationFragment
       }
     }
+    ssrAPIs: allGatsbyApiCall(filter: { group: { eq: "SSRAPI" } }) {
+      group(field: name) {
```

Comment:
It might be worth to put this into fragment as this is duplicated in ssr, browser and node API reference pages (I can help with that)

Reply:
Yeah, will put this into a fragment when I've got time tomorrow (not sure)

Reply:
Done!